crates/cli-support/src/js/mod.rs

```diff
@@ -2634,63 +2635,115 @@ impl<'a, 'b> SubContext<'a, 'b> {
         class_name: &'b str,
         export: &decode::Export,
     ) -> Result<(), Error> {
-        let wasm_name =
-            wasm_bindgen_shared::struct_function_export_name(class_name, &export.function.name);
-
+        let mut fn_name = export.function.name;
+        let wasm_name = struct_function_export_name(class_name, fn_name);
         let descriptor = match self.cx.describe(&wasm_name) {
             None => return Ok(()),
             Some(d) => d,
         };
-
-        let function_name = if export.is_constructor {
-            "constructor"
-        } else {
-            &export.function.name
+        let docs = |raw_docs| format_doc_comments(&export.comments, Some(raw_docs));
+        let getter_from_method_call = |c: &mut ExportedClass, docs, fn_name, g_name| {
```

Comment:
Since this and `setter_from_method_call` are only called once, could their contents be inlined below?

Reply:
This method no longer exists